### PR TITLE
Reword static pod task page

### PR DESCRIPTION
**Pull request by sftim**
Reword https://kubernetes.io/docs/tasks/administer-cluster/static-pod/ and move it to https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/ (where, I think, it belongs)

Split into multiple commits to make it easier to see the rewording.

### CODE REVIEW

**Comment by sftim** [comment]
@jsafrane @jimangel how does this look to you?

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+reviewers:
+- jsafrane
+title: Create static Pods
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+
+*Static Pods* are managed directly by kubelet daemon on a specific node,
```
Comment (steveperry-53) [inline content/en/docs/tasks/configure-pod-container/static-pod.md]: "the kubelet daemon"

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+without the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
+observing it.
```
Comment (steveperry-53) [inline content/en/docs/tasks/configure-pod-container/static-pod.md]: "observing them"

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+Unlike Pods that are managed by the control plane (eg, a
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}};
+instead, kubelet itself watches it and restarts it when it crashes.
```
Comment (steveperry-53) [inline content/en/docs/tasks/configure-pod-container/static-pod.md]: Unlike Pods that are managed by the control plane, static Pods are managed by the kubelet. The kubelet watches static Pods, and restarts them when they crash.

General note: Choose a way of referring to the kubelet, and then be consistent throughout the whole topic. I prefer "the kubelet", but some people use "kubelet". I'd recommend against "Kubelet".

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+There are no health checks for the containers in a static pod.
+
+Static pods are always bound to one {{< glossary_tooltip term_id="kubelet" >}} daemon on a specific node.
```
Comment (steveperry-53) [inline content/en/docs/tasks/configure-pod-container/static-pod.md]: General comment:
Our style is to capitalize, or camel case, the names of Kubernetes objects. So change pod to Pod throughout the topic.

Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: Changed!

**Comment by sftim** [comment]
/assign @tfogo

(think all comments are now addressed)

**Comment by zacharysarah** [comment]
@sftim 👋 Thanks for this! Thanks also for adding the redirect. :sparkles: There's one piece from @steveperry-53's feedback still to be addressed:
> General note: Choose a way of referring to the kubelet, and then be consistent throughout the whole topic. I prefer "the kubelet", but some people use "kubelet". I'd recommend against "Kubelet".

I notice mixed use of "kubelet" and "the kubelet" throughout. I also prefer "the kubelet" and would ask that you standardize accordingly.

Otherwise, please squash your commits when you're ready to merge. 🙇

**Comment by sftim** [comment]
Changed to “the kubelet” or similar throughout.

**Comment by kbhawkey** [comment]
Hi. I am trying to track down a local build problem. Did the static-pod file get moved? I see the source file, but I am having trouble building this file.
Page preview:
https://deploy-preview-14831--kubernetes-io-master-staging.netlify.com/docs/tasks/configure-pod-container/static-pod/

Current location of file on k8s/website:
https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/

**Comment by sftim** [comment]
Hmm, I'm not sure what has happened here. My guess is some GitHub shenanigans, but it could somehow be my error instead.

I expected that https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/ would work and that https://kubernetes.io/docs/tasks/administer-cluster/static-pod/ would redirect there.

**Comment by kbhawkey** [comment]
Hi @sftim. I am seeing a problem with building the file in configure-pod-container/static-pod.md-- cannot find the mirror-pod glossary entry. The mirror-pod glossary file is in the source, but I cannot find the term in the glossary on the current website. Perhaps my eyes are tired.
https://kubernetes.io/docs/reference/glossary?fundamental=true

**Pull request merged**